### PR TITLE
document and release groups and teams

### PR DESCRIPTION
**Pull request by ryanfkeepers**
Adds the release documentation and reveals all cli commands for groups and teams channelMessages and files support.

---

#### Does this PR need a docs update or release note?

- [x] :white_check_mark: Yes, it's included

#### Type of change

- [x] :world_map: Documentation

#### Issue(s)

* #3988 

#### Test Plan

- [x] :muscle: Manual

### CODE REVIEW

File: website/docs/setup/m365-access.md [hunk 1]
```diff
@@ -52,11 +52,18 @@ then click **Add permissions**.
...
+| ChannelSettings.Read.All | Application | Read all Teams' channel settings |
+| Chat.Read.All | Application | Read all Teams' chats and chat messages |
 | Contacts.ReadWrite | Application | Read and write contacts in all mailboxes |
+| Directory.Read.All | Application | Read all organization directory data |
 | Files.ReadWrite.All | Application | Read and write files in all site collections |
 | MailboxSettings.Read | Application | Read all user mailbox settings |
 | Mail.ReadWrite | Application | Read and write mail in all mailboxes |
+| Member.Read.Hidden | Application | Read hidden group memberships |
 | Sites.FullControl.All | Application | Have full control of all site collections |
+| TeamMember.Read.All | Application | Read all Teams' user memberships |
```
Comment (neha-Gupta1) [inline website/docs/setup/m365-access.md:65]: Is this permission getting used currently?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+### Added
+- Groups and Teams service support!  Channel messages and Files are now available for backup and restore in the CLI: `corso backup create groups --group '*'`
+  * The cli commands for "groups" and "teams" can be used interchangably, and will operate on the same backup data.
+  * Restore and Export is available for channel files.  Channel messages are available for Export, but not Restore.
```
Comment (meain) [inline CHANGELOG.md:13]: 2 caveats:
- Export for files is still pending
- As of now only files from public channels are backed up (ie root site)

File: website/docs/setup/m365-access.md (hunk 1 above)
Comment (ryanfkeepers) [reply website/docs/setup/m365-access.md:65]: It's listed on the alcion permissions requirements, and we're copying that set for release.

File: CHANGELOG.md (hunk 2 above)
Comment (vkamra) [reply CHANGELOG.md:13]: These caveats are ok but we should do the following:
1) `Groups and Teams service support!` -> Call it Preview. CLI commands should provide the preview message also.
2) Remove Export.
3) Add the caveats to "Known issues" and limitations. The limitations being: Groups support is in preview and not complete. Files are only backed up from public channels. Export is not yet availbable.

Comment (ryanfkeepers) [reply CHANGELOG.md:13]: > Remove Export
Sharepoint export, you mean?  Or both sharepoint and messages?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+### Added
+- Groups and Teams service support!  Channel messages and Files are now available for backup and restore in the CLI: `corso backup create groups --group '*'`
+  * The cli commands for "groups" and "teams" can be used interchangably, and will operate on the same backup data.
+  * Channel messages are available for Export, but not Restore.
+  * Groups' SharePoint files are available for Restore, but not Export.
```
Comment (vkamra) [inline CHANGELOG.md:14]: Remove from here - called out in limitations
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+### Added
+- Groups and Teams service support!  Channel messages and Files are now available for backup and restore in the CLI: `corso backup create groups --group '*'`
```
Comment (vkamra) [inline CHANGELOG.md:11]:
```suggestion
- Groups and Teams service support available as feature preview!  Channel messages and Files are now available for backup and restore in the CLI: `corso backup create groups --group '*'`
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  * The cli commands for "groups" and "teams" can be used interchangably, and will operate on the same backup data.
+  * Channel messages are available for Export, but not Restore.
+  * Groups' SharePoint files are available for Restore, but not Export.
+  * New permissions are required to backup Channel messages.  See the [Corso Documentation](https://corsobackup.io/docs/setup/m365-access/#configure-required-permissions) for complete details.
+  Even though Channel message restoration is not available, message write permissions are included to cover future integration.
+  * Support is available in an early-access status, and may be subject to breaking changes.
```
Comment (vkamra) [inline CHANGELOG.md:17]:
```suggestion
  * This is a feature preview, and may be subject to breaking changes based on feedback and testing.
```

File: website/docs/support/known-issues.md [hunk 6]
```diff
@@ -26,4 +26,10 @@ Below is a list of known Corso issues and limitations:
 
 * Link shares with password protection can't be restored.
 
-* All replies in a Teams Conversation are removed from backup when the root message gets deleted.
+* Teams conversation replies are only backed up if the parent message is available at the time of backup.
+
+* Groups SharePoint files do not support Export.
```
Comment (vkamra) [inline website/docs/support/known-issues.md:31]:
```suggestion
* Groups SharePoint files do not support Export. This issue will be addressed in a follow-up release.
```

File: website/docs/support/known-issues.md [hunk 7]
```diff
@@ -26,4 +26,10 @@ Below is a list of known Corso issues and limitations:
...
+
+* Teams messages do not support Restore.
```
Comment (vkamra) [inline website/docs/support/known-issues.md:33]:
```suggestion
* Teams messages do not support Restore due to limited Graph API support to create messages.
```

**Review by vkamra — CHANGES_REQUESTED** [review]
A few changes requested

**Comment by ryanfkeepers** [comment]
@vkamra Changes made.

**Review by vkamra — APPROVED** [review]

**Pull request merged**